### ISSUE TITLE
rename vectorize.table "table" parameter

### ISSUE THREAD

**Issue by ChuckHend** [tembo-io/pg_vectorize#145]
```sql
SELECT vectorize.table(
    job_name    => 'product_search_hf',
    "table"     => 'products',
    primary_key => 'product_id',
    columns     => ARRAY['product_name', 'description'],
    transformer => 'sentence-transformers/all-MiniLM-L6-v2',
    schedule    => 'realtime'
);
```

Having "table" as parameter causes a lot of issues for users. If the double quotes are missing the call will fail because `table` is a reserved word. We should rename that perhaps to "table_name" or "relation", and change the datatype to `regclass`. Idea with changing to `regclass` is that we will get more meaningful data validation errors if the table name provided does not exist, and I _think_ using `regclass` would also mean we could remove the "schema" parameter.

**Comment by Abiji-2020** [comment]
@ChuckHend Can i take a look in this Issue ??

**Comment by ChuckHend** [comment]
> [@ChuckHend](https://github.com/ChuckHend) Can i take a look in this Issue ??

@Abiji-2020, yes! I think ideally we are able to change the `table` parameter to something like: `table_name: REGCLASS`. If not, it could be as simple as just renaming it. https://github.com/tembo-io/pg_vectorize/pull/163 is open already, so we may end up splitting the bounty.
